source/includes/environment_config/_30-create.md.erb

```diff
@@ -0,0 +1,119 @@
+## Create an environment
+
+```shell
+$ curl 'https://ci.example.com/go/api/admin/environments/' \
+      -u 'username:password' \
+      -H 'Accept: application/vnd.go.cd.v1+json' \
+      -H 'Content-Type: application/json' \
+      -X POST -d '{
+        "name" : "new_environment",
+        "pipelines" : [
+          {
+          "name" : "up2"
+          }
+        ],
+        "agents" : [
+          {
+            "uuid" : "12345678-e2f6-4c78-123456789012"
+          }
+        ],
+        "environment_variables" : [
+          {
+            "name" : "username",
+            "value" : "admin",
+            "secure" : false
+          }
+        ]
+      }'
+```
+
+> The above command returns JSON structured like this:
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/vnd.go.cd.v1+json; charset=utf-8
+```
+
+```json
+{
+  "_links": {
+    "self": {
+      "href": "https://ci.example.com/go/api/admin/environments/new_environment"
+    },
+    "doc": {
+      "href": "http://api.go.cd/#environment_config"
+    },
+    "find": {
+      "href": "https://ci.example.com/go/api/admin/environments/:environment_name"
+    }
+  },
+  "name": "new_environment",
+  "pipelines": [
+    {
+      "_links": {
+        "self": {
+          "href": "https://ci.example.com/go/api/admin/pipelines/up42"
+        },
+        "doc": {
+          "href": "http://api.go.cd/#pipeline_config"
+        },
+        "find": {
+          "href": "https://ci.example.com/go/api/admin/pipelines/:pipeline_name"
+        }
+      },
+      "name": "up42"
+    }
+  ],
+  "agents": [
+    {
+      "_links": {
+        "self": {
+          "href": "https://ci.example.com/go/api/agents/12345678-e2f6-4c78-123456789012"
+        },
+        "doc": {
+          "href": "http://api.go.cd/#agents"
+        },
+        "find": {
+          "href": "https://ci.example.com/go/api/agents/:uuid"
+        }
+      },
+      "uuid": "12345678-e2f6-4c78-123456789012"
+    }
+  ],
+  "environment_variables": [
+    {
+      "secure": false,
+      "name": "username",
+      "value": "admin"
+    },
+    {
+      "secure": true,
+      "name": "password",
+      "encrypted_value": "LSd1TI0eLa+DjytHjj0qjA=="
+    }
+  ]
+}
+```
+
+Creates an environment
+
+<%= available_since('16.6.0') %>
+
+<p class='http-request-heading'>HTTP Request</p>
+
+`POST /go/api/admin/environments/`
+
+The following properties may be specified:
+
+<%=
+  describe_object nil do
+  name                      'String',  'The environment config name'
+  pipelines                 'Array',   'The list of pipeline names that belongs to the specified environment'
+  agents                    'Array',   'The list of agent uuids that belongs to the specified environment'
```

Comment:
how about "List of pipeline names/agent uuids that should be added to this environment"?